### PR TITLE
Use sleepmode.loft.sh/last-activity instead of sleep mode config when disabled

### PR DESCRIPTION
**Pull request by bkneis**

### CODE REVIEW

File: pkg/workspace/list.go [hunk 1]
```diff
@@ -162,10 +163,18 @@ func listProWorkspaces(ctx context.Context, devPodConfig *config.Config, log log
...
 
 			// last used timestamp
-			lastUsedTimestamp := types.Time{}
+			var lastUsedTimestamp types.Time
 			sleepModeConfig := instance.Status.SleepModeConfig
 			if sleepModeConfig != nil {
 				lastUsedTimestamp = types.Unix(sleepModeConfig.Status.LastActivity, 0)
+			} else {
+				var ts int64
+				if val, ok := instance.Annotations["sleepmode.loft.sh/last-activity"]; ok {
```
Comment (pascalbreuninger) [inline pkg/workspace/list.go:172]: `instance.annotations` could be nil

Comment (pascalbreuninger) [reply pkg/workspace/list.go:172]: other than that LGTM

Comment (bkneis) [reply pkg/workspace/list.go]: good spot :detective:

**Review by pascalbreuninger — APPROVED** [review]
LGTM

**Pull request merged**